content/en/containers/amazon_ecs/apm.md

```diff
@@ -222,7 +222,7 @@ package main
 import (
     "net/http"
     "io/ioutil"
-    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer" // use "github.com/DataDog/dd-trace-go/v2/ddtrace/tracer" if you're using v2.x
```

Comment:
```suggestion
    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer" // 1.x
    // "github.com/DataDog/dd-trace-go/v2/ddtrace/tracer" // 2.x
```

Reply:
Although I didn't get to every code snippet, this suggestion applies to nearly every code snippet